.github/workflows/build.yml

```diff
@@ -5,10 +5,13 @@ on:
   pull_request:
 
 jobs:
-  tests:
+  github_mactest:
     runs-on: macos-latest
     steps:
     - uses: actions/checkout@main
+    - uses: actions/setup-python@v2
+      with:
+          python-version: '3.7'
```

Comment:
should we use the opportunity to test also the minimum required version which is 3.6?